### DISCUSSION TITLE
Command line setting interaction

### DISCUSSION THREAD

**Discussion by gfsiciliani** [KDE/ghostwriter#854] [Q&A]
It appears to me that the only CLI interaction possible with ghostwriter is to open the app and to disable GPU. 

I am new to Linux and CLI and imagined that it may be possible to have more control from the CLI to facilitate things like launching the app with a certain theme or focus mode on. This would be both to serve as storing a sort of default setting but also allow for context-specific cases (i.e. I've written a bash script that starts a journaling file and would like to be able to bash script a theme change).

TL;DR -- is it possible to script theme changes and other settings in ghostwriter?

**Comment by gfsiciliani** [comment]
it seems like development has moved to invent.kde.org and discussions respectively to bugs.kde.org. I'll leave this open here in case others would like to contribute.
